RecoPPS/RPixEfficiencyTools/python/ReferenceAnalysisDQMWorker_cfg.py

```diff
@@ -105,33 +107,39 @@
 runNumber=options.runNumber
 
 
-if runNumber < firstRunOfTheYear:
-    print("This run belongs to before 2018 data taking")
-elif runNumber <= lastRunPreTs1:
-    print("Analyzing Pre-TS1 data")
-elif runNumber <= lastRunPreTs2:
-    print("Analyzing data taken between TS1 and TS2")
-    for i in range(4):
-        if (i == 1 or i == 3):
-            fiducialYLow[i] -= 0.5
-            fiducialYHigh[i] -= 0.5
-        else:
-            fiducialYLow[i] += 0.5
-            fiducialYHigh[i] += 0.5
-elif runNumber <= lastRunOfTheYear:
-    print("Analyzing Post-TS2 data")
-    for i in range(4):
-        if (i == 1 or i == 3):
-            fiducialYLow[i] -= 1
-            fiducialYHigh[i] -= 1
-        else:
-            fiducialYLow[i] += 1
-            fiducialYHigh[i] += 1
-elif runNumber > lastRunOfTheYear:
-    print("This run doesn't belong to 2018 data taking")
+def print_information_about_LHC_run():
```

Comment:
@AndreaBellora I want to know your opinion on printing information about LHC run. The same code is in the EA DQM Worker.
Firstly, this code is not working. Do we want to fix it or delete it? 
Secondly, I can't stand that these workers have duplicated code. I would push to extract this method to a separate file.

Reply:
Let's go ahead and delete this, we don't need it anymore and there's no point in keeping it. Just leave the dummy values for the fiducial* variables and delete this stuff